apps/system/test/marionette/video_test.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
```

Comment:
In gaia, the browser is just a special window in the system. There is not different app for the browser (despite we show an icon), and that's the reason all the browser tests are inside the system. We could potentially group them in a folder for make it more understandable. +1 to the renaming